1-mandatory/2-people-i-know.js

```diff
@@ -394,6 +406,8 @@ This time, I only want the full names ("<firstname> <lastname>") of my friends w
 */
 
 let friendsWhoAreColleaguesOfStacie = [];
```

Comment:
It should be something like this `let friendsWhoAreColleaguesOfStacie = [];
for (const friend of friends) {
  for (const coll of friend.colleagues) {
    if (coll.name === "Stacie Villarreal") {
      friendsWhoAreColleaguesOfStacie.push(
        `${friend.name.first} ${friend.name.last}`
      );
    }
  }
}`